packages/swingset-liveslots/tsconfig.json

```
@@ -5,6 +5,8 @@
     "*.js",
     "scripts/**/*.js",
     "src/**/*.js",
+    "src/**/*.ts",
+    "src/**/*.d.ts",
     "test/**/*.js",
     "tools/**/*.js",
```

Similarly, I think we've aligned on top level folders
```suggestion
    "scripts",
    "src",
    "test",
    "tools",
```